docs/concepts/custom-resources.md

```diff
@@ -73,6 +73,10 @@ The below diagram shows how the `KongPlugin` resource can be linked to an
 
 ## KongClusterPlugin
 
+_This resource requires the `kubernetes.io/ingress.class` annotation. Its value
+must match the value of the controller's `--ingress-class` argument, which is
+"kong" by default._
```

Comment:
```suggestion
_This resource requires the [`kubernetes.io/ingress.class` annotation](../README.md#resource-classes)._
```

The definition of what an ingress class means is already present in the docs. Let's link to it instead of repeating it over and over.

This comment applies to all repetitions of such pattern.

Reply:
> This comment applies to all repetitions of such pattern.

Note that there are two other repetitions in this file still left.